recipes/repository.rb

```diff
@@ -25,6 +25,13 @@
     action :install
   end
 
+  # Trust new APT key
+  execute 'install new apt key' do
+    command 'apt-key adv --recv-keys --keyserver hkp://keyserver.ubuntu.com:80 A2923DFF56EDA6E76E55E492D3A80E30382E94DE'
+    not_if 'apt-key list | grep 382E94DE'
+  end
+
```

Comment:
No need to spend too much time on this. 

I am just a little worried about the "flow" for trusting the new RPM key. It involves multiple commands, with `|` and `grep`. I was wondering if we can be sure that the output will be consistent across all systems using RPM and the different versions of `rpm-import`.